### ISSUE TITLE
`{foo}.txt` should be same as `{foo,}.txt`

### ISSUE THREAD

**Issue by timotheecour4** [facelessuser/wcmatch#192]
```py
from wcmatch import glob
print(glob.globmatch('foo.txt', '{foo,}.txt', flags=glob.GLOBSTAR|glob.BRACE))
print(glob.globmatch('foo.txt', '{foo}.txt', flags=glob.GLOBSTAR|glob.BRACE))
```
prints:
True
False

but should print:
True
True

**Comment by facelessuser** [comment]
Can I ask what your expectations are based on? Wcmatch uses a brace expansion implementation that I wrote called BraceX.  The expansion is based on Bash as stated [here](https://facelessuser.github.io/bracex/#overview). Its results are compared directly against Bash to ensure accuracy. There are a few exceptions in behavior, but this is not one of them. I assume your expectation might be based on some other shell's behavior.  If there is no sequence, braces are treated as braces.

Notice below that the first example matches the file but fails to execute it due to permissions, so the match works! The second case does not match README.md because the braces are then treated as literal braces:

```console
bash-5.1$ ./{README,}.md
bash: ./README.md: Permission denied
bash-5.1$ ./{README}.md
bash: ./{README}.md: No such file or directory
```
We are not targeting ZSH, but let's see what it does. Interestingly we see the same behavior.

```console
➜  wcmatch git:(main) ./{README,}.md
zsh: permission denied: ./README.md
➜  wcmatch git:(main) ./{README}.md
zsh: no such file or directory: ./{README}.md
```

In short, I'm not seeing a bug here.

**Comment by facelessuser** [comment]
I'm going to go ahead and close this as there doesn't appear to be a real bug. Feel free to reopen it if there is more information that can be provided to justify this being a bug.
